config/resque.yml

```diff
@@ -1 +1,2 @@
 development: <%= ENV.fetch('REDIS_URL', 'localhost:6379') %>
+test: <%= ENV.fetch('REDIS_URL', 'localhost:6379') %>
```

Comment:
would we need this for prod as well?

Reply:
no because config/resque.yml is in shared configs.